### PR TITLE
[694] Provide a representation to display representations

### PR DESCRIPTION
**Pull request by gcoutable**
### Type of this PR 

- [ ] Bug fix
- [x] New feature or improvement
- [ ] Documentation
- [ ] Cleanup
- [ ] Test
- [ ] Build/Releng

### Issue(s)

https://github.com/eclipse-sirius/sirius-components/issues/694

### What does this PR do?

It provides a new representation that is able to display all representations available for the selection.

### How to test this PR?

- [ ] Frontend Unit tests
- [ ] Backend Unit tests
- [ ] Cypress : please specify
- [ ] Manual Test : please specify

### Checklist

- [ ] I have read CONTRIBUTING carefully.
- [ ] I have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [ ] All my commits are signed-off (`-s`) with my mail address of my Eclipse Account.
- [ ] I have covered my changes by unit tests or integration tests or manual tests.
- [ ] All tests pass.
- [ ] I have updated the documentation accordingly
- [ ] New React components are availables in storybook

### CODE REVIEW

File: backend/sirius-web-representations/src/main/java/org/eclipse/sirius/web/representations/Success.java [hunk 1]
```diff
@@ -12,11 +12,48 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The status to return when the representation description handler success.
  *
  * @author gcoutable
  */
-public class Success implements IStatus {
+public final class Success implements IStatus {
```
Comment (sbegaudeau) [inline backend/sirius-web-representations/src/main/java/org/eclipse/sirius/web/representations/Success.java:23]: Do not modify that heavily this class. While switching to a builder pattern is a good solution for `CreateAccountInput` a couple days ago since it went from having 3 parameters to having 11 parameters, here you just need one additional parameter. No need to break that many classes for such change. Something like this would have way less impact:

```
public class Success implements IStatus {
  private final Map<String, Object> parameters;

  public Success() {
    this(new HashMap<>());
  }

  public Success(Map<String, Object> parameters) {
    this.parameters = Objects.requireNonNull(parameters);
  }
}
```

On top of that, such change will need to be documented in the CHANGELOD.adoc too. There are 24 files in this PR that should not be impacted by this PR and I'm not counting the other files impacted in additional products.

File: backend/sirius-web-forms/src/main/java/org/eclipse/sirius/web/forms/components/ListComponent.java [hunk 2]
```diff
@@ -32,6 +35,8 @@
  */
 public class ListComponent implements IComponent {
 
+    public static final String ACTION_VARIABLE = "action"; //$NON-NLS-1$
```
Comment (sbegaudeau) [inline backend/sirius-web-forms/src/main/java/org/eclipse/sirius/web/forms/components/ListComponent.java:38]: Is that variable used?

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java [hunk 3]
```diff
@@ -79,9 +86,33 @@ public Diagram create(String label, Object targetObject, DiagramDescription diag
         return newDiagram;
     }
 
+    @Override
+    public boolean shouldRefresh(IEditingContext editingContext, IDiagramContext diagramContext, ChangeDescription changeDescription) {
+        Diagram previousDiagram = diagramContext.getDiagram();
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:91]: that's not the "previous" diagram

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java [hunk 4]
```diff
@@ -79,9 +86,33 @@ public Diagram create(String label, Object targetObject, DiagramDescription diag
...
+        // @formatter:on
+
+        // @formatter:off
+        return optionalDiagramDescription.flatMap(this.representationRefreshPolicyRegistry::getRepresentationRefreshPolicy)
+                .orElseGet(() -> this.defaultShouldRefresh(diagramContext))
+                .shouldRefresh(changeDescription);
+        // @formatter:on
+    }
+
+    private IRepresentationRefreshPolicy defaultShouldRefresh(IDiagramContext diagramContext) {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:105]: `defaultShouldRefresh` -> `getDefaultRefreshPolicy`

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java [hunk 5]
```diff
@@ -79,9 +86,33 @@ public Diagram create(String label, Object targetObject, DiagramDescription diag
         return newDiagram;
     }
 
+    @Override
+    public boolean shouldRefresh(IEditingContext editingContext, IDiagramContext diagramContext, ChangeDescription changeDescription) {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:90]: Why is the DiagramCreationService in charge of the refresh policy of a diagram? I don't see why this code should be in this class.

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventProcessor.java [hunk 6]
```diff
@@ -144,8 +143,7 @@ public void refresh(ChangeDescription changeDescription) {
      * @return <code>true</code> if the diagram should be refreshed, <code>false</code> otherwise
      */
     private boolean shouldRefresh(ChangeDescription changeDescription) {
-        return ChangeKind.SEMANTIC_CHANGE.equals(changeDescription.getKind())
-                || (DiagramChangeKind.DIAGRAM_LAYOUT_CHANGE.equals(changeDescription.getKind()) && changeDescription.getSourceId().equals(this.diagramContext.getDiagram().getId()));
+        return this.diagramCreationService.shouldRefresh(this.editingContext, this.diagramContext, changeDescription);
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventProcessor.java:146]: Do not rely on the diagramCreationService for that

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/api/IDiagramCreationService.java [hunk 7]
```diff
@@ -57,6 +58,24 @@
...
+     * @param editingContext
+     *            The editing context
+     * @param diagramContext
+     *            The diagram context that hold the diagram that could need a refresh
+     * @param changeDescription
+     *            The change description
+     * @return <code>true</code> if the diagram held by the diagram context needs a refresh, <code>false</code>
+     *         otherwise
+     */
+    boolean shouldRefresh(IEditingContext editingContext, IDiagramContext diagramContext, ChangeDescription changeDescription);
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/api/IDiagramCreationService.java:77]: Remove this, do not modify this file

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeEdgeToolOnDiagramEventHandler.java [hunk 8]
```diff
@@ -103,7 +103,6 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
             if (optionalTool.isPresent()) {
                 IStatus status = this.executeTool(editingContext, diagramContext, input.getDiagramSourceElementId(), input.getDiagramTargetElementId(), optionalTool.get());
                 if (status instanceof Success) {
-
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeEdgeToolOnDiagramEventHandler.java:106]: No need to change this file

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeNodeToolOnDiagramEventHandler.java [hunk 9]
```diff
@@ -109,7 +109,6 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
                 IStatus status = this.executeTool(editingContext, diagramContext, input.getDiagramElementId(), optionalTool.get(), input.getStartingPositionX(), input.getStartingPositionY(),
                         input.getSelectedObjectId());
                 if (status instanceof Success) {
-
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeNodeToolOnDiagramEventHandler.java:112]: No need to change this file

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java [hunk 10]
```diff
@@ -147,13 +143,26 @@ public void refresh(ChangeDescription changeDescription) {
...
+    private boolean shouldRefresh(ChangeDescription changeDescription) {
+        // @formatter:off
+        return this.representationRefreshPolicyRegistry.getRepresentationRefreshPolicy(this.formCreationParameters.getFormDescription())
+                .orElseGet(this::defaultShouldRefresh)
+                .shouldRefresh(changeDescription);
+        // @formatter:on
+
+    }
+
+    private IRepresentationRefreshPolicy defaultShouldRefresh() {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java:155]: `defaultShouldRefresh` -> `getDefaultRefreshPolicy`

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/RepresentationsEventProcessorFactory.java [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+    public <T extends IRepresentationEventProcessor> Optional<T> createRepresentationEventProcessor(Class<T> representationEventProcessorClass, IRepresentationConfiguration configuration,
+            IEditingContext editingContext) {
+        if (IFormEventProcessor.class.isAssignableFrom(representationEventProcessorClass) && configuration instanceof RepresentationsConfiguration) {
+            RepresentationsConfiguration representationsConfiguration = (RepresentationsConfiguration) configuration;
+
+            FormDescription formDescription = this.representationsDescriptionProvider.getRepresentationsDescription();
+            Optional<Object> optionalObject = this.objectService.getObject(editingContext, representationsConfiguration.getObjectId());
+            if (optionalObject.isPresent()) {
+                Object object = optionalObject.get();
+                if (formDescription != null) {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/RepresentationsEventProcessorFactory.java:81]: Why test for `null` if the API does not return an optional? No need for this test, `null` is not an acceptable value.

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/FormCreationParameters.java [hunk 12]
```diff
@@ -33,8 +33,6 @@
 
     private UUID id;
 
-    private String label;
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/FormCreationParameters.java:36]: Why are you removing the label?

File: backend/sirius-web-spring-collaborative-selection/src/main/java/org/eclipse/sirius/web/spring/collaborative/selection/SelectionEventProcessor.java [hunk 13]
```diff
@@ -111,6 +117,18 @@ public void refresh(ChangeDescription changeDescription) {
...
 
+    private boolean shouldRefresh(ChangeDescription changeDescription) {
+        // @formatter:off
+        return this.representationRefreshPolicyRegistry.getRepresentationRefreshPolicy(this.selectionDescription)
+                .orElseGet(this::defaultShouldRefresh)
+                .shouldRefresh(changeDescription);
+        // @formatter:on
+    }
+
+    private IRepresentationRefreshPolicy defaultShouldRefresh() {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-selection/src/main/java/org/eclipse/sirius/web/spring/collaborative/selection/SelectionEventProcessor.java:128]: `defaultShouldRefresh` -> `getDefaultRefreshPolicy`

File: backend/sirius-web-spring-collaborative-trees/src/main/java/org/eclipse/sirius/web/spring/collaborative/trees/TreeEventProcessor.java [hunk 14]
```diff
@@ -136,27 +141,37 @@ public void refresh(ChangeDescription changeDescription) {
...
-        }
+    private boolean shouldRefresh(ChangeDescription changeDescription) {
+        // @formatter:off
+        return this.representationRefreshPolicyRegistry.getRepresentationRefreshPolicy(this.treeCreationParameters.getTreeDescription())
+                .orElseGet(this::defaultShouldRefresh)
+                .shouldRefresh(changeDescription);
+        // @formatter:on
+    }
+
+    private IRepresentationRefreshPolicy defaultShouldRefresh() {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-trees/src/main/java/org/eclipse/sirius/web/spring/collaborative/trees/TreeEventProcessor.java:152]: `defaultShouldRefresh` -> `getDefaultRefreshPolicy`

File: backend/sirius-web-spring-collaborative-validation/src/main/java/org/eclipse/sirius/web/spring/collaborative/validation/ValidationEventProcessor.java [hunk 15]
```diff
@@ -138,8 +144,16 @@ public void refresh(ChangeDescription changeDescription) {
...
-        return ChangeKind.SEMANTIC_CHANGE.equals(changeKind);
+    private boolean shouldRefresh(ChangeDescription changeDescription) {
+        // @formatter:off
+        return this.representationRefreshPolicyRegistry.getRepresentationRefreshPolicy(this.validationDescription)
+                .orElseGet(this::defaultShouldRefresh)
+                .shouldRefresh(changeDescription);
+        // @formatter:on
+    }
+
+    private IRepresentationRefreshPolicy defaultShouldRefresh() {
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative-validation/src/main/java/org/eclipse/sirius/web/spring/collaborative/validation/ValidationEventProcessor.java:155]: `defaultShouldRefresh` -> `getDefaultRefreshPolicy`

File: backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/ChangeDescription.java [hunk 16]
```diff
@@ -33,12 +35,18 @@
...
+
     public ChangeDescription(String kind, UUID sourceId, IInput input) {
         this.kind = Objects.requireNonNull(kind);
         this.sourceId = Objects.requireNonNull(sourceId);
         this.input = Objects.requireNonNull(input);
     }
 
+    public Object addParameter(String key, Object value) {
+        return this.parameters.put(key, value);
+    }
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/ChangeDescription.java:48]: Do not create such stateful method, we need to keep immutable classes immutable. instead ask for a Map<> in the constructor. You can create an additional constructor which require this map and make the existing one fallback to the new constructor with an empty map.

File: backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/IRepresentationRefreshPolicy.java [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.api;
+
+/**
+ * The interface used to describe the refresh policy of a representation regarding the given change description.
+ *
+ * @author gcoutable
+ */
+@FunctionalInterface
```
Comment (sbegaudeau) [inline backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/IRepresentationRefreshPolicy.java:20]: No need for that annotation but could you document this new concept in the CHANGELOG.adoc please?

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
I have not reviewed the frontend yet but here are some comments on the backend side

File: backend/sirius-web-representations/src/main/java/org/eclipse/sirius/web/representations/Success.java (hunk 1 above)
Comment (gcoutable) [reply backend/sirius-web-representations/src/main/java/org/eclipse/sirius/web/representations/Success.java:23]: Done

File: backend/sirius-web-forms/src/main/java/org/eclipse/sirius/web/forms/components/ListComponent.java (hunk 2 above)
Comment (gcoutable) [reply backend/sirius-web-forms/src/main/java/org/eclipse/sirius/web/forms/components/ListComponent.java:38]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java (hunk 3 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:91]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java (hunk 4 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:105]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java (hunk 5 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramCreationService.java:90]: I don't remember well why I put this code here. Maybe because it would have added one parameter to the diagram event processor while the parameter was already used in the diagram creation service and the diagram creation service was already in charge of the refresh, so, why not the should refresh.
Anyway, this code has been moved to the diagram event processor.

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventProcessor.java (hunk 6 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/DiagramEventProcessor.java:146]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/api/IDiagramCreationService.java (hunk 7 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/api/IDiagramCreationService.java:77]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeEdgeToolOnDiagramEventHandler.java (hunk 8 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeEdgeToolOnDiagramEventHandler.java:106]: Done

File: backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeNodeToolOnDiagramEventHandler.java (hunk 9 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/handlers/InvokeNodeToolOnDiagramEventHandler.java:112]: Done

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java (hunk 10 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/FormEventProcessor.java:155]: Done

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/RepresentationsEventProcessorFactory.java (hunk 11 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/RepresentationsEventProcessorFactory.java:81]: Done

File: backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/FormCreationParameters.java (hunk 12 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-forms/src/main/java/org/eclipse/sirius/web/spring/collaborative/forms/api/FormCreationParameters.java:36]: Because it has no use except to create a `FormCreationParameters` from another `FormCreationParameters`.

File: backend/sirius-web-spring-collaborative-selection/src/main/java/org/eclipse/sirius/web/spring/collaborative/selection/SelectionEventProcessor.java (hunk 13 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-selection/src/main/java/org/eclipse/sirius/web/spring/collaborative/selection/SelectionEventProcessor.java:128]: Done

File: backend/sirius-web-spring-collaborative-trees/src/main/java/org/eclipse/sirius/web/spring/collaborative/trees/TreeEventProcessor.java (hunk 14 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-trees/src/main/java/org/eclipse/sirius/web/spring/collaborative/trees/TreeEventProcessor.java:152]: Done

File: backend/sirius-web-spring-collaborative-validation/src/main/java/org/eclipse/sirius/web/spring/collaborative/validation/ValidationEventProcessor.java (hunk 15 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative-validation/src/main/java/org/eclipse/sirius/web/spring/collaborative/validation/ValidationEventProcessor.java:155]: Done

File: backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/IRepresentationRefreshPolicy.java (hunk 17 above)
Comment (gcoutable) [reply backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/api/IRepresentationRefreshPolicy.java:20]: Done

File: frontend/src/properties/propertysections/ListPropertySection.types.ts [hunk 18]
```diff
@@ -10,10 +10,27 @@
...
+import { Selection } from 'workbench/Workbench.types';
 
 export interface ListPropertySectionProps {
+  editingContextId: string;
+  formId: string;
   widget: List;
   subscribers: Subscriber[];
   readonly: boolean;
+  setSelection?: (selection: Selection) => void;
+  canBeSelected?: (item: ListItem) => boolean;
```
Comment (sbegaudeau) [inline frontend/src/properties/propertysections/ListPropertySection.types.ts:23]: Are those really optional?

File: frontend/src/representations/RepresentationsWebSocketContainerMachine.ts [hunk 19]
```diff
@@ -0,0 +1,277 @@
...
+        if (isFormRefreshedEventPayload(data.representationsEvent)) {
+          const { form } = data.representationsEvent;
+          let widget = null;
+          const firstPage = [...form.pages].shift();
+          if (firstPage) {
+            const firstGroup = [...firstPage.groups].shift();
+            if (firstGroup) {
+              widget = [...firstGroup.widgets].shift();
+            }
+          }
```
Comment (sbegaudeau) [inline frontend/src/representations/RepresentationsWebSocketContainerMachine.ts:257]: form.pages[0]?.groups[0]?.widgets[0]

File: frontend/src/workbench/RightSite.types.ts [hunk 20]
```diff
@@ -15,5 +15,6 @@ import { Selection } from 'workbench/Workbench.types';
 export interface RightSiteProps {
   editingContextId: string;
   selection?: Selection;
+  setSelection?: (selection: Selection) => void;
```
Comment (sbegaudeau) [inline frontend/src/workbench/RightSite.types.ts:18]: is that really optional?

**Review by sbegaudeau — CHANGES_REQUESTED** [review]

**Comment by sbegaudeau** [comment]
I'll squash those commits and bump the version, could you rebase your PR on the Sirius Web side? thanks

**Comment by gcoutable** [comment]
> 
> 
> I'll squash those commits and bump the version, could you rebase your PR on the Sirius Web side? thanks

Done

**Review by sbegaudeau — APPROVED** [review]

**Comment by sbegaudeau** [comment]
I'll merge it once the build is done
